bindings/c/test/mako/ddsketch.hpp

```diff
@@ -173,6 +181,11 @@ class DDSketchBase {
 		assert(fabs(errorGuarantee - anotherSketch.errorGuarantee) < EPS &&
 		       anotherSketch.buckets.size() == buckets.size());
 		for (size_t i = 0; i < anotherSketch.buckets.size(); i++) {
+			// Check for overflow
+			if ((buckets[i] > 0) && (anotherSketch.buckets[i] > std::numeric_limits<uint32_t>::max() - buckets[i])) {
+				std::cerr << "Overflow detected when merging sketches! Exiting..." << std::endl;
+				std::exit(-1);
```

Comment:
Should we exit? Or should we stop collecting stats?
If you just want to run mako for a very long time and didn't specify sampling rate (which no one really does), I think we should just warn and continue running.

Reply:
Using uint64_t for bucket size should just remove this issue. We would need 2^64 ops at the SAME latency to overflow a bucket -- our max TPS lately is around 900K TPS so it would take 600,000 years to hit this issue 🙂